pandas/core/groupby/groupby.py

```diff
@@ -2383,7 +2384,7 @@ def tail(self, n=5):
         mask = self._cumcount_array(ascending=False) < n
         return self._selected_obj[mask]
 
-    def _reindex_output(self, output):
+    def _reindex_output(self, output, fill_value=np.NaN):
```

Comment:
can you update the doc-string, also can you try to type fill_value (and output & the return if you can, I think Union[Series, DataFrame])